lib/tasks/exchange_rates.rake

```diff
@@ -0,0 +1,104 @@
+namespace :exchange_rates do
+  desc 'Remove and re-build old monthly rates inclusive of the provided data (does not download from XE)'
```

Comment:
Note this Rake task does not pull down rates from XE it uses what rates we have in the DB. This could be changed with an extra argument but not needed for this task